qiskit/providers/ibmq/api/clients/account.py

```diff
@@ -165,11 +165,12 @@ def job_submit(
                     qobj_dict=qobj_dict,
                     job_name=job_name,
                     job_share_level=job_share_level)
-            except Exception:  # pylint: disable=broad-except
+            except Exception as ex:  # pylint: disable=broad-except
                 # Fall back to submitting the Qobj via POST if object storage
                 # failed.
                 logger.info('Submitting the job via object storage failed: '
-                            'retrying via regular POST upload.')
+                            'retrying via regular POST upload: %s',
+                            str(ex))
```

Comment:
Given object storage involves multiple steps, is this sufficient for debugging? Maybe we can log the traceback at debug level?

Reply:
Yes! Added in 19b5f76